packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -6813,11 +6813,13 @@ addEventListener('fetch', event => {});`
 			  [4mhttps://github.com/cloudflare/wrangler2/issues/new/choose[0m
 
 			",
-			  "warn": "[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mHere are the 2 largest dependencies included in your script:[0m
+			  "warn": "[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mYour worker failed validation because it exceeded startup limits.[0m
```

Comment:
```suggestion
			  "warn": "[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mYour Worker failed validation because it exceeded startup limits.[0m
```